src/auth-service/models/Survey.js

```
@@ -0,0 +1,432 @@
+SurveySchema.pre("update", function (next) {
+  return next();
+});
+
+SurveySchema.statics = {
+  async register(args, next) {
+    try {
+      let createBody = args;
+
+      // Remove _id if present
+      if (createBody._id) {
+        delete createBody._id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)